test/functional-tests/stepDefinitions/definitionFileSteps.js

```diff
@@ -36,4 +36,11 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
     Data.caseType = 'Conditionals';
   });
 
+  Given(/^I have filled out the create case filters$/, function () {
```

Comment:
Not sure about the name of this step.

Reply:
Shouldn't we be physically selecting the dropdowns here? Sth like this:

 When(/^I select the 'Case type' drop down option for dynamic filters$/, async function () {
    await searchPage.selectJurisdiction(TestData.jurisdiction);
    await searchPage.selectCaseType(TestData.caseType);
  });